cdap-data-fabric/src/test/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetServiceTestBase.java

```diff
@@ -231,6 +235,7 @@ public void onChange(ServiceDiscovered serviceDiscovered) {
   public void after() throws Exception {
     Services.chainStop(service, opExecutorService, txManager);
     namespaceClient.delete(Id.Namespace.DEFAULT);
+    namespaceStore.delete(Id.Namespace.DEFAULT);
```

Comment:
`namespaceClient.delete(Id.Namespace.DEFAULT)` should delete the namespace, right? Do we need `namespaceStore.delete(Id.Namespace.DEFAULT)` in addition?